Website/src/items/renderings/edgewebsite/Page Content/WelcomeMessage.yml

```diff
@@ -0,0 +1,35 @@
+﻿---
+ID: "d356e8d0-3af5-483b-ae0d-2693ba87ffd1"
+Parent: "3006ec5b-0438-414b-860e-e6754658ea12"
+Template: "04646a89-996f-4ee7-878a-ffdbf1f0ef0d"
+Path: /sitecore/layout/Renderings/Project/edgewebsite/Page Content/WelcomeMessage
```

Comment:
Would it be possible to name this the same way as the other files. i.e. `DynamicWelcomeMessage` ?